### ISSUE TITLE
Nextcloud support

### ISSUE THREAD

**Issue by karlitschek** [prsnbrg/files_audio_converter#1]
Hi, this is a great app idea. Do you plan to build this for Nextcloud? If yes than we could help you with some mentoring. And there is even an app contest running https://nextcloud.com/blog/all-app-developers-put-your-hands-up-best-nextcloud-app-contest/ ;-)
Let me know.
Cheers
Frank

**Comment by prsnbrg** [comment]
@karlitschek 
Yes,
at first I will code this for ownCloud and then I will test it for Nextcloud.

I'll probably need help coding, though. It is my first app for ownCloud/Nextcloud.

**Comment by karlitschek** [comment]
@prsnbrg Sure. Let me know when I can help :-)

**Comment by prsnbrg** [comment]
@karlitschek I see you are in the ownCloud and Nextcloud team, right?

I have no concrete idea how to build a modal by selecting the new file type. How do I build something like this?

**Comment by karlitschek** [comment]
@prsnbrg yes. I'm the founder or ownCloud and Nextcloud. Nowadays most of the community is building Nextcloud. This is why I suggested to contribute there.
